tests/client_tests/go/pg_type_workaround.sql

```diff
@@ -0,0 +1,13 @@
+-- dbext:type=CRATE:host=localhost:port=4200:dbname=doc
+
+
+create table doc.pg_type (
+    oid int,
+    typedelim string,
+    typelem int,
+    typname string,
+    typtype string
```

Comment:
The naming seems odd `typedelim` vs `typname` but that's how it its.

Reply:
oh this was actually a type - guess the typdelim column isn't used by pgx

Reply:
What about `typelem` and `typtype`? they also have a different prefix.

Reply:
Ah, the correct prefix is `typ`.